src/article/dtos/article-output.dto.ts

```diff
@@ -1,21 +1,28 @@
+import { Field, Int, ObjectType } from '@nestjs/graphql';
 import { Expose, Type } from 'class-transformer';
 
 import { AuthorOutput } from './author-output.dto';
 
+@ObjectType('Article')
 export class ArticleOutput {
   @Expose()
+  @Field(() => Int)
   id: number;
 
   @Expose()
+  @Field()
   title: string;
 
   @Expose()
+  @Field()
   post: string;
 
   @Expose()
+  @Field()
   createdAt: Date;
 
   @Expose()
+  @Field()
   updatedAt: Date;
 
   @Expose()
```

Comment:
Shouldn't we have a `@Field()` decorator for the type `AuthorOutput` for `author` , or is it skipped for later? If so then we need to also update `ArticleResolver` to include field resolvers for `author`.

Reply:
Left it for future PR